### ISSUE TITLE
question about forestdb

### ISSUE THREAD

**Issue by calvin2021y** [eBay/Jungle#136]
It must be link with forestdb to work ?

**Comment by greensky00** [comment]
Yes, Jungle uses ForestDB for the indexing of a single Table file.

**Comment by calvin2021y** [comment]
thanks for the explain.

May I ask how much QPS for a nvme device ?

**Comment by greensky00** [comment]
That is not a fixed number and varies widely according to the device's IOPS, workload pattern (random/sequential), locality, key-value size, the entire dataset size, RAM size, the number of concurrent threads, and many others.
